### DISCUSSION TITLE
Is it something wrong?

### DISCUSSION THREAD

**Discussion by javigarcmart** [pratik227/quasar-qgrid#105] [General]
Hi i want to attach this q-grid to my project and there's something wrong I think, could u help me or tell me If I am doing something bad.

The command 
`quasar ext add q-grid`
doesnt work to me.
I am working with Quasar CLI to at version 2 with Vue 3.

But well I'll see that u put on package.json something that u take de version from local 
in line 16.

`"quasar-app-extension-qgrid": "file:../app-extension",`

So I copy the folder with the same path on my project, doesnt work to..

So i dont know What Am I doing wrong..

I copy one of the examples that You Have and don't show me nothing at screen.

Hope u can help Me. thanks :ddd

**Comment by pratik227** [comment]
Hi @javigarcmart,

You are running wrong command.

`quasar ext add qgrid`

Try this.
